crates/core/src/utils.rs

```diff
@@ -1,14 +1,20 @@
+use anyhow::{Context, Result};
```

Comment:
We use anyhow in CLI only. In CLI, we use anyhow for ergonomic error propagation and user-facing reports.
In core/library crates, we define typed errors with `thiserror`.

Reply:
got it I had moved the function from cli to core to reused it so I have to modify that